package.json

```diff
@@ -7,10 +7,12 @@
     "dev:client": "NODE_ENV=development webpack -w",
     "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
     "prod:build": "NODE_ENV=production webpack -p",
-    "format": "prettier --write 'src/**/*.js'",
+    "format": "prettier --write '{src,test}/**/*.js'",
     "lint": "eslint --cache --max-warnings 0",
-    "lint:all": "npm run lint -- 'src/**/*.js'",
-    "test": "npm run lint:all",
+    "lint:all": "npm run lint -- '{src,test}/**/*.js'",
+    "pretest": "npm run lint:all",
+    "test": "mocha '{src,test}/**/*.test.js'",
+    "test:watch": "nodemon -w src -w test -x 'npm test'",
     "precommit": "lint-staged"
   },
   "lint-staged": {
```

Comment:
I would like to run as many tests and checks as we can in the precommit hook. I wasn't sure if calling them from `lint-staged` was correct, however, since `npm test` are the unit tests (not linting) and `npm run lint:all` is executed as a prerequistite to tests (so it doesn't actually need to be called explicitly).